devel/resource_dasm/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+PortGroup               compiler_blacklist_versions 1.0
+
+github.setup            fuzziqersoftware resource_dasm 46ad32e
+github.tarball_from     archive
+
+compiler.blacklist-append {clang < 1300}
+
+version                 2023.03.15
+revision                0
+categories              devel
+maintainers             {alum.wpi.edu:arno+macports @fracai} openmaintainer
+license                 MIT
+
+description             Tools for reverse-engineering classic Mac OS applications and games.
+long_description        {*}${description}
+
+checksums               rmd160  190a5eb9a174d8eafdc17ca736f730e2d12862ec \
+                        sha256  db5e42d9ef9df9d4d40f42fd7c1b67cc2f5108a06e30ae983fd6e40046ad22e0 \
+                        size    457269
+
+set pyver               3.11
+set python_version      [string index ${pyver} 0][string range ${pyver} 2 end]
+
+patchfiles              CMakeLists-txt.diff
+
+depends_build-append    port:phosg \
```

Comment:
phosg should probably be a library dependency (i.e. required at both build time and runtime). I'm guessing the same is true for zlib as well.